diff -Naur a/lib/typescript.js b/lib/typescript.js
--- a/lib/typescript.js	2019-12-25 18:06:10.283603060 +0100
+++ b/lib/typescript.js	2019-12-25 18:06:10.309603280 +0100
@@ -23705,6 +23705,7 @@
         // extra cost of calling `getParseTreeNode` when calling these functions from inside the
         // checker.
         var checker = {
+isTypeIdenticalTo: isTypeIdenticalTo,
             getNodeCount: function () { return ts.sum(host.getSourceFiles(), "nodeCount"); },
             getIdentifierCount: function () { return ts.sum(host.getSourceFiles(), "identifierCount"); },
             getSymbolCount: function () { return ts.sum(host.getSourceFiles(), "symbolCount") + symbolCount; },
