diff -Naur a/lib/typescript.js b/lib/typescript.js
--- a/lib/typescript.js	2019-12-25 18:06:14.466638449 +0100
+++ b/lib/typescript.js	2019-12-25 18:06:14.494638685 +0100
@@ -24458,6 +24458,7 @@
         // extra cost of calling `getParseTreeNode` when calling these functions from inside the
         // checker.
         var checker = {
+isTypeIdenticalTo: isTypeIdenticalTo,
             getNodeCount: function () { return ts.sum(host.getSourceFiles(), "nodeCount"); },
             getIdentifierCount: function () { return ts.sum(host.getSourceFiles(), "identifierCount"); },
             getSymbolCount: function () { return ts.sum(host.getSourceFiles(), "symbolCount") + symbolCount; },
