diff -Naur a/lib/typescript.js b/lib/typescript.js
--- a/lib/typescript.js	2019-12-25 19:35:54.838363019 +0100
+++ b/lib/typescript.js	2019-12-25 19:35:54.869363268 +0100
@@ -32189,6 +32189,7 @@
         // extra cost of calling `getParseTreeNode` when calling these functions from inside the
         // checker.
         var checker = {
+isTypeIdenticalTo: isTypeIdenticalTo, isTypeAssignableTo: isTypeAssignableTo,
             getNodeCount: function () { return ts.sum(host.getSourceFiles(), "nodeCount"); },
             getIdentifierCount: function () { return ts.sum(host.getSourceFiles(), "identifierCount"); },
             getSymbolCount: function () { return ts.sum(host.getSourceFiles(), "symbolCount") + symbolCount; },
