diff -Naur a/lib/typescript.js b/lib/typescript.js
--- a/lib/typescript.js	2020-07-04 22:44:51.875617652 +0200
+++ b/lib/typescript.js	2020-07-04 22:44:51.928618030 +0200
@@ -34409,6 +34409,7 @@
         // extra cost of calling `getParseTreeNode` when calling these functions from inside the
         // checker.
         var checker = {
+isTypeIdenticalTo: isTypeIdenticalTo, isTypeAssignableTo: isTypeAssignableTo,
             getNodeCount: function () { return ts.sum(host.getSourceFiles(), "nodeCount"); },
             getIdentifierCount: function () { return ts.sum(host.getSourceFiles(), "identifierCount"); },
             getSymbolCount: function () { return ts.sum(host.getSourceFiles(), "symbolCount") + symbolCount; },
